scripts/migrate_test_runs.py

```diff
@@ -0,0 +1,52 @@
+import sys
+from django.core.exceptions import ObjectDoesNotExist
+from squad.core.models import Project, Build, Environment
+from squad.core.tasks import UpdateProjectStatus
+
+
+def run(*args):
+
+    parsed_args = dict(arg.split("=") for arg in args if '=' in arg)
+
+    if 'old_project_slug' not in parsed_args.keys():
+        print("ERROR: old_project_slug missing")
+        sys.exit(1)
+    if 'new_project_slug' not in parsed_args.keys():
+        print("ERROR: new_project_slug missing")
+        sys.exit(1)
+    if 'env_slug' not in parsed_args.keys():
+        print("ERROR: env_slug missing")
+        sys.exit(1)
```

Comment:
in a management command this would be handled by the framework, you just need to declare the arguments